commit 0111be24a2b99481049584b793076c35b67d77f4
Author: Gawain Lynch <gawain.lynch@gmail.com>
Date:   Fri Aug 15 08:25:06 2014 +0100

    Correctly break; when looping on $response->placesLived when a primary location is defined

diff --git a/hybridauth/Hybrid/Providers/Google.php b/hybridauth/Hybrid/Providers/Google.php
index 50d287a..dc1e2fb 100644
--- a/hybridauth/Hybrid/Providers/Google.php
+++ b/hybridauth/Hybrid/Providers/Google.php
@@ -118,6 +118,7 @@ class Hybrid_Providers_Google extends Hybrid_Provider_Model_OAuth2
 					if($c->primary == true){ 
 						$this->user->profile->address 	= $c->value;
 						$this->user->profile->city 	= $c->value;
+						break;
 					}
 				}else{
 					if(property_exists($c,'value')){
